api/axis.py

```diff
@@ -334,7 +340,7 @@ def _checkout(self, title_id, patron_id, internal_format):
         response = self.request(url, data=args, method="POST")
         return response
 
-    def fulfill(self, patron, pin, licensepool, **kwargs):
+    def fulfill(self, patron, pin, licensepool, internal_format, **kwargs):
```

Comment:
`internal_format` was always being passed in, but up to this point it was ignored because the only way to fulfill a book was to return its ACS fulfillment URL (if the Axis 360 API gave you one) or its Findaway fulfillment data (if the API gave you that).

Now we're in a situation where the Axis 360 API may give you _both_ an ACS fulfillment URL _and_ a BookVaultUUID, but only one of the two will actually work. To know which one will work, `fulfill` needs to know which format we told Axis 360 to use when we borrowed the book. That information is kept in `internal_format`.